### PR TITLE
Documentation rewrite

### PR DESCRIPTION
**Pull request by d2dyno**
Based on input from https://github.com/PAhelper/PhoenixAdult.bundle/issues/233, this PR aims to streamline a few things:

1. How quickly a user can understand the manual searching/matching process.
2. Using universal, consistent terminology and definitions.
3. Having a single metadata point, for all currently supported sites, in a non-confusing manner.

Currently, I have only started the work. For my own sake, here are the items I want to accomplish:

- [x] Finish writing Manual search usage
- [x] Write simple terms for each level of search capability, linking back to their full meaning.
- [x] For any document that gets too long, separate them into their own files (currently thinking: `sitelist.md`, `manualsearch.md` `filenaming.md`)
- [x] Rewrite Site list to match the above rewrites - Pending level 3 only
- [x] Proof-read sitelist.md for consistency and clarity
- [x] Add search types for sites with none listed so far

### CODE REVIEW

**Comment by d2dyno** [comment]
@claygoldfinch any thoughts on the [manualsearch.md](https://github.com/d2dyno/PhoenixAdult.bundle/blob/master/docs/manualsearch.md) document? Going for clear definitions, as well as easy first-timer readability. Once I am confident this document is at a good starting point, I will overhaul the site list.

**Comment by d2dyno** [comment]
I have now shortened the main readme, to provide basic information, but to ultimately link to the new docs. Please let me know if I skimmed the readme too much and some portion does not make sense.

**Comment by claygoldfinch** [comment]
Hell yeah, this looks awesome. I'll do a more thorough review tomorrow, but super impressed so far. Nice work!

**Comment by claygoldfinch** [comment]
I've had a chance to look over this, and I think there's a few changes I'd make:

1. Let's combine SceneID Search and SceneID match into a single term, "SceneID Match". While they are technically different, I don't think it's going to effect the average user much.
2. Likewise, let's combine Date Search and Date Match into a single term, "Date Match". Same idea as above, plus I don't know of any sites who actually support Date Search.

Another thing I want to note (for the sake of clarity), though this doesn't necessarily have to be added to the document: Adding a date to a search term will never cause an issue, as long as the date is immediately after the Site Name. If that is the case, the date will be stripped from the search term and analyzed long before the search term is plugged into it's associated site. For example (and I picked DirectURL as it's the most picky about finding a match):

"SiteName - Date - DirectURL" No issues with finding a match
"SiteName - DirectURL - Date" Will have issues matching
"SiteName - Scene Title - DirectURL" Will have issues matching
"Site Name - DirectURL - Scene Title" Will have issues matching

That said, whether or not adding a date will actually improve your search results is entirely dependent on the site.

Based on all the info above, I think this is how I would go about doing the whole "Levels":

**Level One: Enhanced Search**
- Title and/or Actor Search available
  - Date Match/SceneID Match (if a SceneID is available) will improve matching

**Level Two: Limited Search**
- Title and/or Actor Search available
  - No Date Match/SceneID Match

Level Three: Exact Matching
- SceneID Only
  - Date can be added before SceneID, but as the agent is basically just plugging the SceneID into a URL only one match will be found.
  - Title/Actor/Whatever can be added after the SceneID, but again only one scene is actually being found so it's not actually gonna "improve" matching (though the code may boost the score of the one result available)
- Direct URL Only (Let's use this name as it's similar to SceneID Only)
  - Date can be added before the URL, but (like SceneID) will not actually "improve" matching
  - Nothing can be added after the URL as it will cause issues

Also, let's set Date Add as it's own thing, outside of the Level system. Only 2-3 sites support it, and I think it's different enough that people may want to know when specific sites support it.

**Comment by Sinjin69** [comment]
I'm not a fan of the levels nomenclature.
I do like, Enhanced Search, Limited Search, and Exact Match

**Comment by claygoldfinch** [comment]
Enhanced/Limited/Exact works for me! Like you said, I think the main goal is just to simplify terminology, and it does just that.

**Comment by Sinjin69** [comment]
I think simplifying is great. My problem is that the level numbers just feel arbitrary. I was left wondering, does Level 3 mean that I inherit Level 2 and 1, or does LLevel 1 mean the best.

I how it was referred to as Enhanced, Limited, and Exact. Hopefully that will avoid any hierarchical connotations of using numbers.

**Comment by d2dyno** [comment]
I updated the terminology, and adjusted examples and definitions to reflect date match/sceneid match merge with search terms. I feel the [Limited Search](https://github.com/d2dyno/PhoenixAdult.bundle/blob/master/docs/manualsearch.md#limited-search) section is a bit barren, but I am not really sure anything else fits in there. I have one or two more style changes to make, but this document is ready other than that I think. Will comb through the site list next.

**Comment by d2dyno** [comment]
Site list updated. I would like to have these resolved before moving to merge:

Missing search types:
```
#### - Karups
#### - Private
#### - SexArt
#### - SinsVR
#### - TeenMegaWorld Network
#### - Tonights Girlfriend
#### - VRBangers
```

Not sure what to categorize these as:
```
#### - Bratty Sis *SceneID (BrattySis - id) *Date Search
#### - Hot Crazy Mess *SceneID Match (HotCrazyMess - id) *Date Search
#### - NewSensations *Actress Search *DVD Search (exact spelling)
#### - NF Busty *shoot ID (NFBusty - id) *Date search
#### - Nubiles.net *Shoot ID (NubilesNet - id) *Date search
#### - Nubiles-porn.com Network *Shoot ID (Nubilesporn - id) *Date search
#### - Nubilesfilms *shoot ID (Nubilesfilms - id) *Date search
#### - Swallowed / TrueAnal / Nymphos *SceneID Only *Date Match
#### - That Sitcom Show *SceneID (ThatSitcomShow - id) *Date Search
```

File: docs/manualsearch.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
++ **Date Match:** Date can be entered alongside other search terms to improve search results
+
+## Limited Search
+#### Limited-search available.
++ Available search methods:
+  - **Title**
+  - **Actor**
+
+## Exact Match
+#### No search available.
```
[the remark this answers is not in the document]
Comment (d2dyno) [reply docs/manualsearch.md]: I revised this a little bit, for readability:
```
#### No search available.
*Locating the correct scene is entirely dependent on entering the correct SceneID or Direct URL. In some cases, you may add additional terms to your search. However, these will not increase the possibility of locating the correct scene and are only implemented for convenience.*
+ **SceneID**
  - Can add the the Date (before the SceneID)
  - Can typically add a Title/Actor (after the SceneID).
+ **Direct URL**
  - Can add the the Date (before the URL)
  - Adding any terms, such as Title/Actors, after the URL will cause issues with matching.

File: docs/sitelist.md [hunk 2]
```diff
@@ -0,0 +1,763 @@
...
+-   Mommy Got Boobs
+-   Moms in Control
+-   Pornstars Like It Big
+-   Racks and Blacks
+-   Real Wife Stories
+-   Shes Gonna Squirt
+-   Teens Like It Big
+-   Teens Like It Black
+-   ZZ Series
+#### - Bratty Sis *SceneID (BrattySis - id) *Date Search
```
[the remark this answers is not in the document]
Comment (d2dyno) [reply docs/sitelist.md]: By title supported, does that mean it enhances the search, or is an alternate method? Just want to be sure

**Comment by d2dyno** [comment]
Updated per comments. The only thing left on my list is the sites which are missing any search type (in the previous comment).

Also thanks @claygoldfinch for all the time to work with me to make this all clear and accurate!

**Comment by d2dyno** [comment]
In an effort to also preserve history, but still shorten the readme, I moved all the old changes to their own file, CHANGELOG.md, which is in most official git specs.

Should the developers ever want, you can also use some neat tools to create 'versions' with this file, here is one I was looking at https://github.com/github-changelog-generator/Github-Changelog-Generator/blob/master/CHANGELOG.md

**Comment by claygoldfinch** [comment]
**Additions:**

- MetArt and MetArtX under label "MetArt Network". Also, put already-supported sites Viv Thomas and SexArt under this label. Limited Search.
- Little Caprice - Enhanced Search
- WowGirls - Limited Search, Date Add
- VIPissy - Limited Search, Date Add
- GirlsOutWest - DirectURL
- Girls Rimming - DirectURL
- Gangbang Creampie - Enhanced Search
- DadCrush - SceneID, Date Add
- Step Secrets - Enhanced Search
- VRHush - DirectURL

**Comment by claygoldfinch** [comment]
Move all these sites under one label, "Other Nubiles Sites". SceneID Only.
  - Anilos
  - Bratty Sis
  - Hot Crazy Mess
  - NF Busty
  - Nubilesfilms
  - NubilesNet
  - That Sitcom Show

File: docs/sitelist.md [hunk 3]
```diff
@@ -0,0 +1,802 @@
...
+  - AllInternal
+  - AssTraffic
+  - CumForCover
+  - PerfectGonzo
+  - PrimeCups
+  - PurePOV
+  - SpermSwap
+  - TamedTeens
++ #### PervMom | Matching type: *[Exact](./manualsearch.md#exact-match)* - **DirectURL**
++ #### Porndoe Premium Network | Matching type: *[Limited](./manualsearch.md#limited-search)*
```
Comment (claygoldfinch) [inline docs/sitelist.md]: Enhanced.

File: docs/sitelist.md [hunk 4]
```diff
@@ -0,0 +1,802 @@
...
+  - Special Feet Force
+  - STG
+  - The White Boxxx
+  - Transbella
+  - Trans Taboo
+  - Tu Venganza
+  - Xchimera
+  - XXX Omas
+  - XXXShades
++ #### PornFidelity | Matching type: *[Limited](./manualsearch.md#limited-search)*
```
Comment (claygoldfinch) [inline docs/sitelist.md]: Enhanced.

File: docs/sitelist.md [hunk 5]
```diff
@@ -0,0 +1,802 @@
...
++ #### PornPros Network | Matching type: *[Exact](./manualsearch.md#exact-match)*
```
Comment (claygoldfinch) [inline docs/sitelist.md]: Might want to add "DirectURL"

File: docs/sitelist.md [hunk 6]
```diff
@@ -0,0 +1,802 @@
...
+  - JurassicCock
+  - MassageCreep
+  - MilfHumiliation
+  - PimpParade
+  - PornPros
+  - RealExGirlfriends
+  - ShadyPi
+  - SquirtDisgrace
+  - TeenBFF
++ #### Other PornPros sites | Matching type: *[Exact](./manualsearch.md#exact-match)*
```
Comment (claygoldfinch) [inline docs/sitelist.md]: Might want to add "DirectURL"

File: docs/sitelist.md [hunk 7]
```diff
@@ -0,0 +1,802 @@
...
+  - Lubed
+  - MyVeryFirstTime
+  - NannySpy
+  - Passion-HD
+  - POVD
+  - PureMature
+  - SpyFam
+  - Tiny4k
++ #### PrettyDirty | Matching type: *[Enhanced](./manualsearch.md#enhanced-search)*
++ #### Private
```
Comment (claygoldfinch) [inline docs/sitelist.md]: Limited

File: docs/sitelist.md [hunk 8]
```diff
@@ -0,0 +1,802 @@
...
+  - Tranny Surprise
+  - VIP Crew
+  - We Live Together
+  - Wives in Pantyhose
++ #### SexArt
++ #### SexyHub | Matching type: *[Limited](./manualsearch.md#limited-search)* - **Title only**
++ #### Screwbox | Matching type: *[Limited](./manualsearch.md#limited-search)*
++ #### SexArt | Matching type: *[Limited](./manualsearch.md#limited-search)*
++ #### SexBabesVR | Matching type: *[Exact](./manualsearch.md#exact-match)*
++ #### SinsVR
```
Comment (claygoldfinch) [inline docs/sitelist.md]: SceneID Only

File: docs/sitelist.md [hunk 9]
```diff
@@ -0,0 +1,802 @@
...
++ #### SisLovesMe | Matching type: *[Exact](./manualsearch.md#exact-match)* - **SceneID** (title + date supported to improve score)
++ #### Spizoo | Matching type: *[Limited](./manualsearch.md#limited-search)*
+  - FirstClassPOV
+  - IntimateLesbians
+  - JessicaJaymesXXX
+  - PornGoesPro
+  - PornstarTease
+  - RawAttack
+  - TheStripperExperience
++ #### StasyQ VR
```
Comment (claygoldfinch) [inline docs/sitelist.md]: SceneID Only

File: docs/sitelist.md [hunk 10]
```diff
@@ -0,0 +1,802 @@
...
+  - TeenJoi
+  - Teen Pies
+  - Teens Do Porn
+  - Teens Love Anal
+  - Teens Love Money
+  - Teeny Black
+  - The Real Workout
+  - This Girl Sucks
+  - Titty Attack
++ #### TeenFidelity | Matching type: *[Limited](./manualsearch.md#limited-search)*
```
Comment (claygoldfinch) [inline docs/sitelist.md]: Enhanced.

File: docs/sitelist.md [hunk 11]
```diff
@@ -0,0 +1,802 @@
...
+  - Teen Stars Only
+  - Teens 3 Some
+  - TmwVRnet
+  - Tricky Masseur
+  - WOW Orgasms
+  - Watch Me Fucked
+  - X Angels
+  - Teen Sex Movs
++ #### That Sitcom Show | Matching type: *[Exact](./manualsearch.md#exact-match)* - **SceneID**
++ #### Tonights Girlfriend
```
Comment (claygoldfinch) [inline docs/sitelist.md]: Limited (Actor Search).

File: docs/sitelist.md [hunk 12]
```diff
@@ -0,0 +1,802 @@
...
+  - Teen Pies
+  - Teens Do Porn
+  - Teens Love Anal
+  - Teens Love Money
+  - Teeny Black
+  - The Real Workout
+  - This Girl Sucks
+  - Titty Attack
++ #### TeenFidelity | Matching type: *[Limited](./manualsearch.md#limited-search)*
++ #### TeenMegaWorld Network
```
Comment (claygoldfinch) [inline docs/sitelist.md]: Enhanced.

File: docs/sitelist.md [hunk 13]
```diff
@@ -0,0 +1,802 @@
...
++ #### Twistys | Matching type: *[Exact](./manualsearch.md#exact-match)* - **SceneID** (title supported to improve score)
+  - Mom Knows Best
+  - Twistys
+  - Twistys Hard
+  - When Girls Play
++ #### Viv Thomas | Matching type: *[Limited](./manualsearch.md#limited-search)* - **Title only**
++ #### VRBangers
++ #### VirtualRealPorn | Matching type: *[Limited](./manualsearch.md#limited-search)* - **Title only**
++ #### VirtualTaboo | Matching type: *[Limited](./manualsearch.md#limited-search)* - **Title only**
++ #### Vixen | Matching type: *[Limited](./manualsearch.md#limited-search)*
```
Comment (claygoldfinch) [inline docs/sitelist.md]: Enhanced.

**Comment by claygoldfinch** [comment]
I think this is about set. You may want to add in a sentence or two re: Date Add, as I didn't see it covered anywhere. Basically just say "Some sites doesn't make release dates available, so the agent will strip the date from your filename/search term instead". Other than that, everything looks good!

File: docs/sitelist.md [hunk 14]
```diff
@@ -0,0 +1,802 @@
...
++ #### TrenchcoatX | Matching type: *[Limited](./manualsearch.md#limited-search)*
++ #### Tushy | Matching type: *[Enhanced](./manualsearch.md#enhanced-search)*
++ #### TushyRaw | Matching type: *[Exact](./manualsearch.md#exact-match)*
++ #### Twistys | Matching type: *[Exact](./manualsearch.md#exact-match)* - **SceneID** (title supported to improve score)
+  - Mom Knows Best
+  - Twistys
+  - Twistys Hard
+  - When Girls Play
++ #### Viv Thomas | Matching type: *[Limited](./manualsearch.md#limited-search)* - **Title only**
++ #### VRBangers
```
[the remark this answers is not in the document]
Comment (claygoldfinch) [reply docs/sitelist.md]: Also this is out of order, alphabetically.

File: docs/sitelist.md [hunk 15]
```diff
@@ -0,0 +1,802 @@
...
+  - Sneaky Sex
+  - Street BlowJobs
+  - Team Squirt
+  - Teens Love Huge Cocks
+  - Top Shelf Pussy
+  - Tranny Surprise
+  - VIP Crew
+  - We Live Together
+  - Wives in Pantyhose
++ #### SexArt
```
[the remark this answers is not in the document]
Comment (d2dyno) [reply docs/sitelist.md]: Great catch!

File: docs/sitelist.md (hunk 14 above)
[the remark this answers is not in the document]
Comment (d2dyno) [reply docs/sitelist.md]: Good catch again! Likely happened because `sort` will put capitals above non-capital.

**Comment by d2dyno** [comment]
Updated per all requests. I did notice 'ManyVids' still has no category, and it seems we skipped updating this one `NewSensations *Actress Search *DVD Search (exact spelling)`

**Comment by claygoldfinch** [comment]
ManyVids is SceneID Only.

New Sensations is kinda in a state of disrepair (working on it), but let's call it DirectURL only.

**Comment by d2dyno** [comment]
Ready to merge.

**Pull request merged**